### PR TITLE
Feature: add function calling quickstart notebook

### PR DESCRIPTION
**Pull request by snova-petrojm**
Add simple notebook that shows how to use function calling with Llama 3.1 to evaluate a mathematical expression

### CODE REVIEW

File: quickstart/function_calling_calculate.ipynb [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    "# Import libraries\n",
+    "import json\n",
+    "import re\n",
+    "from openai import OpenAI\n",
+    "from dotenv import load_dotenv\n",
+    "import os\n",
+    "\n",
+    "# Constants and Configuration\n",
+    "load_dotenv()\n",
+    "API_KEY = os.getenv(\"SAMBANOVA_API_KEY\")\n",
```
Comment (snova-kwasia) [inline quickstart/function_calling_calculate.ipynb:43]: Please change these to match our usual naming for consistency ie
SAMBANOVA_API_KEY
SAMBANOVA_BASE_URL

File: quickstart/function_calling_calculate.ipynb [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+    "BASE_URL = \"https://fast-api.snova.ai/v1\"\n",
```
Comment (snova-kwasia) [inline quickstart/function_calling_calculate.ipynb:44]: This base url is incorrect, we should be using the one from the cloud api not fast-api. Please change

File: quickstart/function_calling_calculate.ipynb [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+    "MODEL = \"Meta-Llama-3.1-70B-Instruct\"\n",
+    "\n",
+    "# OpenAI client setup\n",
```
Comment (snova-kwasia) [inline quickstart/function_calling_calculate.ipynb:47]: Can we not name them directly please? Maybe just add "Client Setup"

**Review by snova-kwasia — CHANGES_REQUESTED** [review]

File: quickstart/function_calling_calculate.ipynb (hunk 2 above)
Comment (snova-petrojm) [reply quickstart/function_calling_calculate.ipynb:44]: Good catch, thanks

**Review by snova-kwasia — APPROVED** [review]
LGTM @snova-petrojm thanks

**Pull request merged**